src/components/FrontChat/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { CSSProperties } from 'react'
+
+import useChat from '../../hooks/useChat'
+
+const styles: {
+  button: CSSProperties
+} = {
+  // Add widget styles here
+  button: {
+    // Add button styles here
+    position: 'fixed',
+    zIndex: 999,
```

Comment:
Could you leave a comment regarding the z-index that's currently used in the core library? It can be helpful to have that reference in the event of future updates by Front.